src/metabase/config.clj

```diff
@@ -23,6 +23,7 @@
    ;; Jetty Settings. Full list of options is available here: https://github.com/ring-clojure/ring/blob/master/ring-jetty-adapter/src/ring/adapter/jetty.clj
    :mb-jetty-port          "3000"
    :mb-jetty-join          "true"
+   :mb-jetty-request-header-size  "8192"
```

Comment:
We'd like to not specify the default here and not set the value if it's not overridden, that way if the default changes in the Jetty library in the future, we don't need to update this value. I'll provide an example.